api/http/management.go

```diff
@@ -95,6 +122,32 @@ func (api *ManagementAPI) GetConfiguration(c *gin.Context) {
 		return
 	}
 
+	// RBAC
+	if len(c.Request.Header.Get(model.RBACHeaderInvetoryGroups)) > 1 {
+		allowed := false
+		for _, group := range strings.Split(
+			c.Request.Header.Get(model.RBACHeaderInvetoryGroups), ",") {
+			allowed, err = api.App.AreDevicesInGroup(
+				ctx, []string{devID.String()}, group)
```

Comment:
just a remark, since there is not much we can do about it: there is a second wave of increase in the API calls that originate from the other API calls, we really call a lot to process only one single client call (client meaning from the outside). and now for each group we call in another service the inventory. do you think we should one day take a closer look at it?

Reply:
in the mean time I added a BET agenda item.